rundeckapp/grails-app/assets/scss/custom/jobs.scss

```diff
@@ -107,3 +112,38 @@
     border-color: #e8e8e8;
   }
 }
+
+
+@media (min-width: 993px) {
+  .execution-aux-info.flex-item-1 {
+    max-width: calc(50vw - 115px);
+  }
+}
+
+.argstring-scrollable {
+  overflow: hidden;
+  word-break: break-all;
+  text-overflow: ellipsis;
+}
+
+.argstring-scrollable .text-secondary {
+  display: block;
+}
+
+.argstring-scrollable .optkey {
+  display: inline-block;
+}
+
+.argstring-scrollable .optvalue {
+  overflow: hidden;
+  word-break: break-all;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  display: block;
+}
+
+// eventargs is typically a table > td
+// found here: /rundeckapp/grails-app/assets/scss/custom/table-events.scss
+.eventargs .argstring-scrollable {
+  display: none;
```

Comment:
It shouldn't hide the arg string in the activity tables.  instead perhaps use `overflow-x: hidden` ?

Reply:
True, this should be overflow-x:hidden. Let me test this and I'll push a change.

Reply:
@jessemarple any luck? I can put something together if you are busy.